.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: node_js
 node_js:
   - "5"
-  - "4"
   - "4.2"
```

Comment:
I think this should be 4.4.2. Sorry, but I have noticed our base docker image is based on 4.4.2.